bigbluebutton-tests/playwright/whiteboard/drawShape.js

```
@@ -0,0 +1,36 @@
+const { MultiUsers } = require("../user/multiusers");
+const { snapshotComparison } = require('./util');
+const e = require('../core/elements');
+const { ELEMENT_WAIT_LONGER_TIME } = require("../core/constants");
+
+class DrawShape extends MultiUsers {
+  constructor(browser, context) {
+    super(browser, context);
+  }
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)